src/pages/permissions/PermissionIdentityGroups.tsx

```diff
@@ -0,0 +1,193 @@
+  const [searchParams] = useSearchParams();
+  const {
+    data: groups = [],
+    error,
+    isLoading,
+  } = useQuery({
+    queryKey: [queryKeys.lxdGroups],
+    queryFn: fetchLxdGroups,
+  });
+
+  if (error) {
+    notify.failure("Loading lxd groups failed", error);
+  }
+
+  const headers = [
+    { content: "Group name", sortKey: "name" },
+    { content: "Description", className: "description" },
+    { "aria-label": "Actions", className: "u-align--right actions" },
+  ];
+
+  const filters: PermissionLxdGroupsFilterType = {
+    queries: searchParams.getAll(QUERY),
+  };
+
+  const isPanelOpen = searchParams.has("panel");
+  const identityGroupsLookup = new Set(identity.groups);
+  const groupsForIdentity = groups.filter((group) => {
+    // only display groups associated to the current identity
+    if (identityGroupsLookup.has(group.name)) {
+      return (
+        filters.queries.every(
+          (q) =>
+            group.name.toLowerCase().includes(q) ||
+            group.description.toLowerCase().includes(q),
+        ) || isPanelOpen
+      );
+    }
+
+    return false;
+  });
+
+  const rows = groupsForIdentity.map((group) => {
+    return {
+      name: group.name,
+      className: "u-row",
+      columns: [
+        {
+          content: (
+            <Link
+              className="u-truncate"
+              title={`name: ${group.name}`}
+              to={`/ui/permissions/lxd-group/${group.name}`}
+            >
+              {group.name}
+            </Link>
+          ),
+          role: "cell",
+          "aria-label": "Name",
+        },
+        {
+          content: (
+            <span className="u-truncate" title={`desc: ${group.description}`}>
+              {group.description}
+            </span>
+          ),
+          role: "cell",
+          "aria-label": "Description",
+        },
+        {
+          className: "actions u-align--right",
+          content: (
+            <PermissionIdentityDeleteGroupBtn
+              key={group.name}
+              idendity={identity}
+              group={group}
+            />
+          ),
+          role: "cell",
+          "aria-label": "Actions",
+        },
+      ],
+      sortData: {
+        name: group.name,
+        description: group.description,
+      },
+    };
+  });
+
+  const { rows: sortedRows, updateSort } = useSortTableData({ rows });
+
+  if (isLoading) {
+    return <Loader text="Loading groups" />;
+  }
+
+  const identityGroupsPanel = panelParams.panel === panels.identityGroups && (
+    <EditIdentityGroupsPanel selectedIdentities={[identity]} />
+  );
+
+  return groupsForIdentity.length > 0 ? (
+    <>
+      <div className="permission-identity-groups">
+        <div className="upper-controls-bar">
+          {!isPanelOpen && (
+            <div className="search-box-wrapper">
+              <PermissionLxdGroupsFilter />
+            </div>
+          )}
+          <PermissionIdentityEditGroupsBtn identities={[identity]} />
```

Comment:
Yeah that does make sense since this is the only possible action on this page :+1:

Reply:
Actually, on second thought this would not be consistent with rest of the ui. The default appearance are applied consistently for buttons that appears after user interact with a selectable table.

Reply:
I meant to comment on the button on the detail page, I think there we have it always visible and it would be the primary option.

Reply:
Aaaaah yes that makes sense, I will pass button props down for this :+1: